### PR TITLE
fix(@desktop): use identicon ring/letter instead of identicon

### PR DESCRIPTION
**Pull request by osmaczko**
fixes: https://github.com/status-im/status-desktop/issues/5149 #5326

- removed identicon related code
- used identicon ring/letter in missing places

### Affected areas
almost everything related to identicons

### Screenshots

image
image

### CODE REVIEW

File: src/app_service/service/accounts/service.nim [hunk 1]
```diff
@@ -140,7 +136,7 @@ proc prepareAccountJsonObject(self: Service, account: GeneratedAccountDto, displ
   result = %* {
     "name": if displayName == "": account.alias else: displayName,
     "address": account.address,
-    "identicon": account.identicon,
+    "identicon": "",
```
Comment (jrainville) [inline src/app_service/service/accounts/service.nim:139]: Can we remove that property here?

File: src/app_service/service/chat/dto/chat.nim [hunk 2]
```diff
@@ -37,7 +37,7 @@ type ChatDto* = object
   members*: seq[ChatMember]
   #membershipUpdateEvents*: seq[ChatMembershipEvent]  ???? It's always null and a question why do we need it here within this context ????
   alias*: string
-  identicon*: string
+  identicon*: string # is it identicon or image???
```
Comment (jrainville) [inline src/app_service/service/chat/dto/chat.nim:40]: should this be renamed `icon`?

File: src/app_service/service/message/dto/pinned_message_update.nim [hunk 3]
```diff
@@ -9,7 +9,7 @@ type PinnedMessageUpdateDto* = object
   messageId*: string
   localChatId*: string # not sure what's this and do we need this at all
   pinnedBy*: string
-  identicon*: string
+  identicon*: string # is it identicon or image???
```
Comment (jrainville) [inline src/app_service/service/message/dto/pinned_message_update.nim:12]: Same thing here

**Review by jrainville — APPROVED** [review]
Great job!

**Comment by osmaczko** [comment]
> Great job!

Thanks for the review. Changes applied. Renamed ambiguous `identicon` fields to `icon` and also removed not needed json properties.

File: src/app/modules/main/chat_section/color_hash_model.nim [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+type
+  ModelRole {.pure.} = enum
+    SegmentLength = UserRole + 1
+    ColorId
+
+QtObject:
+  type
+    Model* = ref object of QAbstractListModel
+      items*: seq[Item]
```
Comment (saledjenic) [inline src/app/modules/main/chat_section/color_hash_model.nim:13]: @osmaczko can we have `items` as private?

**Review by saledjenic — APPROVED** [review]
@osmaczko nice, approved.

**Review by iurimatias — APPROVED** [review]

File: src/app/modules/main/chat_section/color_hash_model.nim (hunk 4 above)
Comment (osmaczko) [reply src/app/modules/main/chat_section/color_hash_model.nim]: Yeah sure, no point to make it public, good catch, thanks.

**Pull request merged**